### ISSUE TITLE
How to use GPU acceleration for LightGBM?

### ISSUE THREAD

**Issue by RashidBakirov** [microsoft/SynapseML#1895]
I could not find any documentation on this. Is GPU acceleration always on?

**Comment by imatiach-msft** [comment]
I don't believe GPU binaries are packaged currently, so I don't think it will work out of the box without some packaging changes and possibly code changes

**Comment by vishalovercome** [comment]
Is there a plan to release it however? The documentation creates the impression that no work needs to be done by developers. 

If there's no plan to release it then can you document the steps to leverage it? Should #1062 be merged as well?

**Comment by ranadeepsingh** [comment]
Thank you for asking. As maintainer @imatiach-msft noted in the thread, GPU binaries are not currently packaged with SynapseML's LightGBM integration, so GPU acceleration is not enabled out of the box. Tracking for future GPU support is ongoing in PR #1062. We are closing this support question as answered.
